Sources/SKCore/CompilationDatabase.swift

```diff
@@ -68,10 +68,55 @@ public func tryLoadCompilationDatabase(
   directory: AbsolutePath,
   _ fileSystem: FileSystem = localFileSystem
 ) -> CompilationDatabase? {
-  // TODO: Support fixed compilation database (compile_flags.txt).
-  return try? JSONCompilationDatabase(directory: directory, fileSystem)
+  return
+    (try? JSONCompilationDatabase(directory: directory, fileSystem))
+    ?? (try? FixedCompilationDatabase(directory: directory, fileSystem))
 }
 
+/// Fixed clang-compatible compilation database (compile_flags.txt).
+///
+/// Each line in the file becomes a command line argument. Example:
+/// ```
+/// -xc++
+/// -I
+/// libwidget/include/
+/// ```
+///
+/// See https://clang.llvm.org/docs/JSONCompilationDatabase.html under Alternatives
+public struct FixedCompilationDatabase: CompilationDatabase, Equatable {
+  public var allCommands: AnySequence<Command> { AnySequence([]) }
+  
+  private let fixedArgs: [String]
+
+  public subscript(path: URL) -> [Command] {
+    [Command(directory: "", filename: path.path, commandLine: fixedArgs + [path.path])]
```

Comment:
We should make Command.directory optional and use `nil` here.

Edit: actually, should we instead store the directory from the path we use when constructing the FixedCompilationDatabase (i.e. the directory that contains compile_commands.txt)?